pom.xml

```diff
@@ -66,7 +66,7 @@
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 		<project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
 		
-		<spring-boot.version>2.2.0.RELEASE</spring-boot.version>
+		<spring-boot.version>2.2.1.RELEASE</spring-boot.version>
```

Comment:
Is there a known issue that caused the problem that required an upgrade to 2.2.1 rather than 2.2.0? If so, please could you add a link to it to the commit message?

Reply:
I have not found any issues that seem relevant to why custom-rel-entites results in an invalid json in the spring-boot project, nor fixes and dependency updates in the 2.2.1 release notes.

Reply:
Agreed - looked into it briefly myself, and couldn't work out the cause of the issue either. Thanks.